debian/control

```diff
@@ -11,7 +11,7 @@ Rules-Requires-Root: binary-targets
 
 Package: asl3
 Architecture: all
-Depends: ${misc:Depends}, asl3-asterisk, asl3-menu, dahdi-linux, lsb-release, curl
+Depends: ${misc:Depends}, asl3-asterisk, asl3-menu, dahdi-linux, bind9-dnsutils, curl, lsb-release, uuid
```

Comment:
The `host` command is actually provided by the `bind9-host` package, which is a dependency of `bind9-dnsutils`. Would it be sufficient to just install `bind9-host`?
Also, I don't see anything in the script using `uuid`. Was this a previous miss or why is it here?

Reply:
Yes, minimally the bind9-host package would be sufficient.  I opted for bind9-dnsutils because it also brings in `nslookup` and `dig`.  If you feel strongly, I'm happy to switch the dependency.

As for "uuid", yes it was a previous miss that affected the `asl-telemetry` command.